.github/workflows/build.yml

```
@@ -64,6 +64,20 @@ jobs:
       - uses: actions/setup-python@v2
       - name: Install Dependencies
         run: sudo ./.github/workflows/posix-deps-apt.sh
+      - name: Add ccache to PATH
+        run: |
+          echo "PATH=/usr/lib/ccache:$PATH" >> $GITHUB_ENV
+      - name: Configure ccache action
+        uses: hendrikmuhs/ccache-action@v1
+      - name: Check Autoconf version 2.69 and aclocal 1.16.3
+        run: |
+          set -e
+          grep "Generated by GNU Autoconf 2.69" configure
+          grep "runstatedir" configure
+          grep "PKG_PROG_PKG_CONFIG" aclocal.m4
```

Personal preference; less noise in the output. The other two `grep`s are fine as they are, IMO.

```suggestion
          grep -q "runstatedir" configure
          grep -q "PKG_PROG_PKG_CONFIG" aclocal.m4
```